### PR TITLE
new package: dash 0.5.10

### PR DESCRIPTION
**Pull request by a-schaefers**
## Checklist

- [x] Latest upstream version.
- [x] I agree to maintain this package (**required**).
- [x] I agree to notify KISS if I can no longer maintain this package (**required**).

### CODE REVIEW

File: community/dash/depends [hunk 1]
```diff
@@ -0,0 +1 @@
+make
```
Comment (dylanaraps) [inline community/dash/depends]: This file can be removed.

File: community/dash/build [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+CC="gcc -static" ./configure --prefix=/usr
```
Comment (dylanaraps) [inline community/dash/build]: Change this to the other style. `export CC` and "expanded" `configure`.

```sh
export CC="gcc -static"

./configure \
    --prefix=/usr
```

**Comment by a-schaefers** [comment]
done

File: community/dash/sources [hunk 3]
```diff
@@ -0,0 +1 @@
+http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.10.tar.gz
```
Comment (dylanaraps) [inline community/dash/sources]:
```sh
http://gondor.apana.org.au/~herbert/dash/files/dash-0.5.10.2.tar.gz
```

This looks to be the latest version. :+1:

File: community/dash/build [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+export CC="gcc -static"
```
Comment (konimex) [inline community/dash/build]: Can this be removed? Since Clang exists in `extra` and is a valid `$CC` option.

Comment (dylanaraps) [reply community/dash/build]: I agree though we'd need to use another method of making builds static.

Comment (konimex) [reply community/dash/build]: `-static` is a `CFLAGS` option so it should be `export CFLAGS="$CFLAGS -static"` should one insist on making a static package.

Comment (dylanaraps) [reply community/dash/build]: Lovely. I'm fine with this change. :+1:

@a-schaefers we may need to go back and fix any packages I may have merged which also set `CC`. I'm happy to do this myself if you like. It's my fault. :)

Comment (a-schaefers) [reply community/dash/build]: let me see what I can do and get back to you. I'm very stubborn about keeping our base be it busybox/gnucoreutils/sbase/ubase/awk/gawk  etc.. all static tools with musl clib. Didn't think about clang.

File: community/dash/sources (hunk 3 above)
Comment (a-schaefers) [reply community/dash/sources]: k

**Comment by a-schaefers** [comment]
New PRs for the aforementioned already merged. Will  fix the pending ones now. Dash is ready.

**Pull request merged**